### DISCUSSION TITLE
Apache Drill

### DISCUSSION THREAD

**Discussion by SakshiTharkude** [apache/superset#20153] [Q&A / Help]
Does apache drill give the feature of drilldown ?

**Comment by ziggekatten** [comment]
The name is confusing. Drill is a data abstraction layer, enabling datasets spanning various physical datasets, as well as an sql API towards textfiles, JSON etc. So, it's not giving drilldowns per se.

**Reply by SakshiTharkude** [reply]
Okay...So can i suggest for this feature 
As drilldown would make this tool more user friendly

**Reply by ziggekatten** [reply]
There are some "drilldown" functions in Superset, by using "emit filter" on the charts. By doing so, clicking on a chart with filter the other charts with the value you clicked on

**Reply by SakshiTharkude** [reply]
Where can i find this emit filter option in which visualization type ?
